src/wp-includes/global-styles-and-settings.php

```diff
@@ -225,6 +225,39 @@ function wp_get_global_stylesheet( $types = array() ) {
 	return $stylesheet;
 }
 
+/**
+ * Gets the global styles custom css from theme.json.
+ *
+ * @since 6.2.0
+ *
+ * @return string Stylesheet.
+ */
+function wp_get_global_styles_custom_css() {
+	// Ignore cache when `WP_DEBUG` is enabled, so it doesn't interfere with the theme developers workflow.
+	$can_use_cached = ! WP_DEBUG;
+	$cache_key      = 'gutenberg_get_global_custom_css';
```

Comment:
Similar to the feedback from @hellofromtonya, this should be `wp_get_global_styles_custom_css` to match the function name. Also the remaining Gutenberg prefix here needs to be removed either way :)